Example/Tests/Matchers/CDTQContainsAllElementsMatcher.h

```diff
@@ -0,0 +1,18 @@
+//
+//  CDTQContainsAllElementsMatcher.h
+//  CloudantQueryObjc
+//
+//  Created by Rhys Short on 21/01/2015.
+//  Copyright (c) 2015 Cloudant Inc. All rights reserved.
+//
+
+#ifndef CloudantQueryObjc_CDTQContainsAllElementsMatcher_h
+#define CloudantQueryObjc_CDTQContainsAllElementsMatcher_h
+
+#import "Expecta.h"
+
+EXPMatcherInterface(containsAllElements, (NSArray * expected));
+
+#define containAllElements containsAllElements
```

Comment:
Is this the standard expecta way to do this? I assume this is here to make the case where one says `expect(blah, to(containAllElements(blah)` to make sense?